pkgs/tools/security/tracee/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenvNoCC, lib, fetchzip, autoPatchelfHook, elfutils, zlib }:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
Wouldn't it be better to use `buildGoModule`?

Reply:
Ive tried building from source and it's been a nightmare because its all designed to be built within their dev containers.
This derivation just patches the prebuilt release so stdenvNoCC.mkDerivation is good.


For a source build I'll almost certainly be using buildGoModule